packages/provider/src/provider.ts

```diff
@@ -479,7 +479,10 @@ const hexlifyTransaction = (
   }
   ethers.utils.checkProperties(transaction, allowed)
 
-  const result: { [key: string]: any } = {}
+  const result: { [key: string]: any } = {
+    delegateCall: (transaction as any).delegateCall,
```

Comment:
say `(transaction as any).delegateCall` is undefined.. on result we will be creating keys with undefined values...

should we instead do like..

```ts
if ((transaction as any).delegateCall) {
  result.delegateCall = (transaction as any).delegateCall
}
// etc..
```

thoughts?